tests/api.c

```diff
@@ -51497,7 +51497,7 @@ static int test_wolfSSL_X509_max_altnames(void)
 #if !defined(NO_FILESYSTEM) && !defined(NO_CERTS) && !defined(NO_RSA)
 
     /* Only test if max alt names has not been modified */
-#if WOLFSSL_MAX_ALT_NAMES == 128
+#if WOLFSSL_MAX_ALT_NAMES == 1024
```

Comment:
Could this be `#if WOLFSSL_MAX_ALT_NAMES <= 1024`?

Reply:
I guess I didnt think about somebody lowering the limit for some reason, yes that should definitely work I will make the change.